codec/codectools/token_test.go

```diff
@@ -0,0 +1,82 @@
+package codectools
+
+import (
+	"strings"
+
+	"github.com/ipld/go-ipld-prime"
+	"github.com/ipld/go-ipld-prime/fluent"
+	"github.com/ipld/go-ipld-prime/must"
+	basicnode "github.com/ipld/go-ipld-prime/node/basic"
+)
+
+var tokenFixtures = []struct {
+	value    ipld.Node
```

Comment:
it would be good to include `name string` here, like `name: "ScalarString"`, so that these can be run as properly named subtests later on.

a codebase with lots of table-driven tests is much harder to maintain and test if the test cases aren't named and run via sub-tests :)